openlibrary/plugins/openlibrary/sentry.py

```diff
@@ -1,26 +1,14 @@
-import logging
-
-import sentry_sdk
-
 import infogami
 from infogami.utils import delegate
+from openlibrary.utils.sentry import Sentry
 
-from openlibrary.plugins.openlibrary.status import get_software_version
-
-logger = logging.getLogger("openlibrary.sentry")
-
-
-def is_enabled():
-    return hasattr(infogami.config, 'sentry') and infogami.config.sentry.enabled
+sentry = None  # type: Sentry
```

Comment:
Not for this PR but for when we finally upgrade to type hints, I believe this will become:
```
from typing import Optional
sentry = None  # type: Optional[Sentry]
```
https://docs.python.org/3/library/typing.html#typing.Optional